com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -179,6 +179,20 @@ internal void Init(VolumeComponent target, Editor inspector)
             OnEnable();
         }
 
+
+        class ParameterSorter : Comparer<(GUIContent, int, SerializedDataParameter)>
+        {
+            public override int Compare((GUIContent, int, SerializedDataParameter) x, (GUIContent, int, SerializedDataParameter) y)
+            {
+                if (x.Item2 < y.Item2)
+                    return -1;
+                else if (x.Item2 == y.Item2)
```

Comment:
Should probably use named parameters instead of `Item2` to keep the code clean and readable.